carservice/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from django.core.validators import MinValueValidator
+from django.contrib.auth.models import User
+
+
+class Car(models.Model):
+    serial_number = models.CharField(max_length=17)
+    car_mileage = models.PositiveIntegerField()
+    car_model = models.CharField(max_length=30)
+    car_year = models.DateField()
+
+    def __str__(self):
+        return f'Model: {self.car_model}, serial number:({self.serial_number})'
+
+
+class Offer(models.Model):
+    description = models.TextField()
+    price = models.FloatField(validators=[MinValueValidator(10.0)])
+
+    car = models.ForeignKey(Car, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return f'Price: ({self.price}), car:({self.car})'
+
+
+class Rent(models.Model):
+    status = models.BooleanField(default=True)
```

Comment:
to nie jest najlepsza nazwa, skoro to boolean to powinien mówić do czego konkretnie się odnosi, bo co to niby znaczy, że status wynosi False? Np. is_available, is_ongoing i tego typu nazwy już mówią co to jest

Reply:
podbijam